src/tutorial/src/step-11/description.md

```diff
@@ -45,7 +45,7 @@ Then, we can use the component in the template as:
 <div class="html">
 
 ```js
-import ChildComp from './ChildComp.vue'
```

Comment:
Not sure why you're suggesting `.js` instead. The tutorial includes a file for `.vue` file.

Reply:
Because there is not a SFC transformer if you use Vue.js in a `.html` file directly. Any `.vue` file must be processed by the building system (vite, webpack etc.), then will be transformed into component's JavaScript and CSS file. If you use Vue.js in HTML, without a building system, you would rather import the JavaScript file that define the *Vue* object directly. If you click the `reset` button, you may see that the "CORRECT" code imports a `.js` file instead of a `.vue` file.

Reply:
I believe this change is correct.

From a process perspective, generally it's preferred to put unrelated changes in separate PRs so they can be reviewed and merged independently. I'd be happy to merge this extension change, but I can't because it's in the same PR as another change that needs further tweaking.

Reply:
Thannks for your advice. I'll pay attention next time.